client/extensions/woocommerce/style.scss

```diff
@@ -5,6 +5,8 @@
 	}
 }
 
+@import 'components/form-dimensions-input/style';
```

Comment:
Once I merge in #13770 (already approved), it'll just be import statements in here.